src/Libraries/RevitNodes/GeometryConversion/UnitConverter.cs

```diff
@@ -12,6 +12,10 @@ public static class UnitConverter
     {
         public static double DynamoToHostFactor(ForgeTypeId specTypeId)
         {
+            if (!UnitUtils.IsMeasurableSpec(specTypeId))
+            {
+                return 1;
```

Comment:
better to `return 1.0;` since return type is double.

Reply:
By the way, I think it may be better to move this check to the caller `SetParameter`, and leave this function just getting the factor. But if there are too many place to modify, we can keep it as is for now.